includes/Shopping/Shopify_Query.php

```diff
@@ -297,4 +304,35 @@ public function get_search( string $search_term ) {
 
 		return $results;
 	}
+
+	/**
+	 * Parse sort type into array for product query
+	 *
+	 * @since 1.21.0
+	 *
+	 * @param string $sort_by sort condition for product query.
+	 * @return array
+	 */
+	protected function parse_sort_by( string $sort_by ): array {
+		
+		switch ( $sort_by ) {
+			case 'a-z':
+				$order = 'TITLE|false';
```

Comment:
This should be an array.

Reply:
```suggestion
				$order = array( 'sort_key' => 'TITLE', 'reverse' => false );
```

Reply:
It gets split into an array at the end ... wasn't sure if repeating array keys for all the cases would be better or not.

Reply:
I think it makes it cleaner this way.

Reply:
I'll update to just use an array